### DISCUSSION TITLE
Turning with KV plough

### DISCUSSION THREAD

**Discussion by coatsy35** [Courseplay/Courseplay_FS22#2750] [Ideas]
As per your issue on plough turn @Tensuko using a twin track crawler tractor, that plough mostly gets the front land wheel jammed on the tractor whilst turning if the turn is towards the wheel, it's the worst combination of vehicle and implement in the game IMO, but of ten given out in contracts. however it's the most realistic combination IRL. You've probably already solved that but thought it worth a mention as whatever you come up with for the articulated will likely fix the twin track as well.

https://github.com/Courseplay/Courseplay_FS22/issues/2748#issue-1880564021

**Comment by Tensuko** [comment]
Wait until we start with that and then I will test combinations and see how well this will be.
It is just a try to improove the turns by reduce the needed turn radius.
What plow is it exactly?

**Comment by coatsy35** [comment]
Same plough as your screenshot KV PW 100 -12 with JD 8RT 340. If you turn right with it down it can't turn as the front wheel jams on the tractor. Lemken plough of different design works better. But as you said, keeping the turn animation centred until just before it's about to be lowered would fix it. IRL when I used a semi mount plough, I would pickup as I come out, turn it vertical and then turn my loop (or fishtail depending) and then just before I pull back in straight I would turn the plough the rest of the way over ready to enter work, occasionally I might have to stop and pause whilst that happened before setting off straight. 

![Screenshot 2023-09-05 130632](https://github.com/Courseplay/Courseplay_FS22/assets/122838698/80e2b7cd-c889-442f-9f69-1768a7d5e61a)

**Reply by coatsy35** [reply]
Forgot to add with a mounted plough you would never usually drive a loop/lightbulb turn, you would perform a fishtail turn/3 point turn, keeps the headlands smaller and allows a tighter turn as long as you have clearance behind youto the hedge so the plough didn't hit it.

**Reply by Tensuko** [reply]
Thats what CP already does..... Just don't panic....

**Comment by coatsy35** [comment]
This new plough functionality is great! Thank you CP team. Just been playing with it, I do like my ploughing :D

**Reply by Tensuko** [reply]
Jep, I now also like to use the lenken more often. 6 headlands are enough now :)

**Reply by coatsy35** [reply]
The Lemken Titan is my favourite. The KV PW100 drives me mad 🤣
